### PR TITLE
payment tests and validators

### PR DESCRIPTION
**Pull request by marcelomileris**

### CODE REVIEW

File: loanmanagement/wsgi.py [hunk 1]
```diff
@@ -8,9 +8,9 @@
 """
 
 import os
-from dj_static import Cling
```
Comment (fnscoder) [inline loanmanagement/wsgi.py]: We need Cling to deploy

File: loanmanagement/wsgi.py [hunk 2]
```diff
@@ -8,9 +8,9 @@
...
+
 from django.core.wsgi import get_wsgi_application
 
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'loanmanagement.settings')
 
-application = Cling(get_wsgi_application())
```
Comment (fnscoder) [inline loanmanagement/wsgi.py]: We need Cling to deploy

File: loanmanagement/settings.py [hunk 3]
```diff
@@ -59,7 +48,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': ['templates'],
```
Comment (fnscoder) [inline loanmanagement/settings.py:52]: Are we going to use templates?

File: core/serializers.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        )
+
+class PaymentCreateSerializer(ModelSerializer):
+    class Meta:
+        model = Payment
+        fields = (
+            'payment',
+            'date',
+            'amount'
+        )
```
Comment (fnscoder) [inline core/serializers.py]: Add a blank line at the end of file

File: .gitignore [hunk 5]
```diff
@@ -3,4 +3,7 @@
 .idea
 *.sqlite3
 *pyc
-__pycache__
\ No newline at end of file
+__pycache__
+venv
+.vscode
+.vscode/
```
Comment (fnscoder) [inline .gitignore]: Add a blank line at the end of file

File: loanmanagement/settings.py (hunk 3 above)
Comment (marcelomileris) [reply loanmanagement/settings.py:52]: I added a squad screen to the django index

**Pull request closed without merging**